### DISCUSSION TITLE
2024.10.0でメールアドレス無しの入会がエラーになります

### DISCUSSION THREAD

**Discussion by fujisan-sky** [misskey-dev/misskey#14735] [Q&A] [answered]
こちらに入れさせて頂くの初めてです。急いで、ご報告すべきかと思い、あまり調べずに入れさせていただいています。
問題があったら申し訳ありません。

昨日リリースされた2024.10.0 を入れさせて頂いたのですが。
メールアドレス無しの場合の、Signupのシーケンスでの最後のSingin処理でエラー[問題が発生しました]になるようです。
MkSignupDialog.form.vue
の最後のあたりにある
			const res = await misskeyApi('signin-flow', {
				username: username.value,
				password: password.value,
			});
			emit('signup', res);
からのsigninです。
（Singin時にCaptchaが追加された影響かと思い、試しにSingup時のCaptchaの値を入れてみたのですが、
同じ値で認証２回は拒否されるようですね。）

**Comment by samunohito** [comment]
@syuilo @kakkokari-gtyih @anatawa12 
https://github.com/misskey-dev/misskey/issues/14699 で議論されていた内容に関係ありますかね…？

**Reply by kakkokari-gtyih** [reply]
あまりないきがする

**Comment by kakkokari-gtyih** [comment, marked as answer]
アカウント登録自体は成功しているはずですので、通常のログイン画面から改めてログイン操作をすることで回避可能です。

代替の仕様については #14736 にて検討します
